### PR TITLE
Add court section

### LINKED ISSUE #12: minha-vez | add court list on lobby
## Introduction
We are going to show a list court

## Implementation
1. Add court Card
i. players name
2. remove the empty case description

### PR DESCRIPTION
**Pull request by frgiovanna**
## Motivation

Closes #12 

This PR adds court section on the lobby page

## Screenshot

Screenshot 2024-12-05 at 15 04 19

### CODE REVIEW

File: src/shared/types/index.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export type Player = { name: string; id: string };
```
Comment (frgiovanna) [inline src/shared/types/index.ts:1]: I tried to define a generic player type that can be used in multiple places as the queue, courts, and on the your-turn page, do you think the server can return that?

File: src/pages/Lobby/components/CourtsSection/types.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { Player } from '@shared/types';
+
+type Court = Player[];
+
+export interface CourtsSectionProps {
+  courts: Court[];
```
Comment (frgiovanna) [inline src/pages/Lobby/components/CourtsSection/types.ts:6]: Do you think this type makes sense? I tried to define something that would work for one or more courts

File: src/shared/components/Court/Court.tsx [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import { Wrapper } from './styles';
+import { CourtProps } from './types';
+
+export function Court({ children }: CourtProps) {
+  return <Wrapper>{children}</Wrapper>;
```
Comment (Wendlereis) [inline src/shared/components/Court/Court.tsx:5]: **Question:** Is it possible to use the `Card` component?

File: src/pages/Lobby/components/CourtsSection/types.ts (hunk 2 above)
Comment (Wendlereis) [reply src/pages/Lobby/components/CourtsSection/types.ts:6]: I agree with this modeling! Congrats

File: src/shared/types/index.ts (hunk 1 above)
Comment (Wendlereis) [reply src/shared/types/index.ts:1]: I can return the player like this. Lets move forward

File: src/shared/components/Court/Court.tsx (hunk 3 above)
Comment (frgiovanna) [reply src/shared/components/Court/Court.tsx]: Let me know if you like this: https://github.com/Wendlereis/minha-vez/pull/32/commits/e2e539ce3265c68c1e169b0efa3d6892e8276f58

Comment (Wendlereis) [reply src/shared/components/Court/Court.tsx]: Cool!

**Pull request merged**